src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala

```diff
@@ -12,6 +12,7 @@ import common.Change
 import common.TreeTraverser
```

Comment:
Just a general comment regarding the commit message: "Innards" are what you can see here (https://www.google.at/search?q=innards&client=firefox-b-ab&source=lnms&tbm=isch&sa=X&ved=0ahUKEwj0t7K5i-TNAhWBRhQKHdpkA-4Q_AUICCgB&biw=1920&bih=1032)... you mean internals I guess ;-)